src/buttons/leave.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  ActionRowBuilder,
+  ButtonInteraction,
+  EmbedBuilder,
+  SelectMenuOptionBuilder,
+  StringSelectMenuBuilder,
+} from "discord.js";
+import { languages } from "../types/help-roles";
+
+module.exports = {
+  name: "join",
```

Comment:
Use the proper name. 
```suggestion
  name: "leave",
```